release.json

```
@@ -22,6 +22,24 @@
         "WINDOWS_DDNPM_SHASUM": "4ca09e6a029dc1db2f2313b6a1ebf770ee3e733dcf500c770e9cd154490d0435",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.25.1-rc.2": {
+        "INTEGRATIONS_CORE_VERSION": "7.25.0",
+        "OMNIBUS_SOFTWARE_VERSION": "7.25.0",
+        "OMNIBUS_RUBY_VERSION": "7.25.0",
+        "JMXFETCH_VERSION": "0.41.0",
+        "JMXFETCH_HASH": "a0b545bdd38308d4d55b49e2c986c4d32fe1388d9bb265f16c1ea4fc545731e3",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.6",
+        "MACOS_BUILD_VERSION": "6.25.1-rc.2"
```

I did not create that tag :P You can either create it on datadog-agent-macos-build or
```suggestion
        "MACOS_BUILD_VERSION": "7.25.1-rc.2"
```